src/js/profile/text.js

```diff
@@ -0,0 +1,20 @@
+import {Component} from "../utils";
+
+export class Text extends Component {
```

Comment:
Is this component re-using the Indicator html blocks? I see you're deleting an element with a `.bar-chart` class